learning/inference.py

```diff
@@ -7,16 +7,19 @@
 from contextlib import contextmanager
 from functools import partial
 from math import radians
-from pathlib import Path
+from pathlib import Path, PurePath
```

Comment:
Why `PurePath`?

Reply:
I don't think I pushed the changes yet but it is used for video creation in inference.